hornetq-core-client/src/main/java/org/hornetq/core/message/impl/MessageImpl.java

```diff
@@ -256,7 +256,7 @@ public void copyHeadersAndProperties(final MessageInternal msg)
       expiration = msg.getExpiration();
       timestamp = msg.getTimestamp();
       priority = msg.getPriority();
-      properties = msg.getTypedProperties();
+      properties = new TypedProperties(msg.getTypedProperties());
```

Comment:
I don't think this is fixing your issue (if you have an issue on copying during the Bridge).

As this is only affecting LargeMessage.copy(id)

Reply:
Unless the issue is with LargeMessages